data/colors/vars/component_dark.ts

```diff
@@ -167,5 +167,24 @@ export default {
       activeBg: alpha(get('scale.red.4'), 0.24),
       hoverText: get('scale.red.3')
     }
+  },
+
+  switchTrack: {
+    bg: get('scale.black'),
+    border: get('scale.gray.4'),
```

Comment:
Is this new border color one that we want to share with _all_ form controls? Like `input` and `select`? Thinking about the contrast ratio for our standard `input` border..

Reply:
I was worried about making that decision in the scope of this PR. These changes just support the ToggleSwitch component, and I think we'll want to zoom out and look at all form inputs.

Here's what I proposed to Allie and Chelsea:
<img width="616" alt="Screen Shot 2022-03-21 at 4 17 53 PM" src="https://user-images.githubusercontent.com/2313998/159356736-f1a1fd36-840a-4d2d-9483-689fb4fc96c0.png">

Reply:
Thanks for that context! That makes sense to me as well. If you add a more generic token _now_ without actually rolling it out to more form components or documenting it, I could see that being confusing for folks utilizing tokens in other apps. Renaming and re-mapping tokens later isn't a huge lift 😄